server/plugin.go

```diff
@@ -81,13 +85,27 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 	p.API.RegisterCommand(getCommand())
-	user, err := p.API.GetUserByUsername(config.Username)
+
+	botId, ensureBotError := p.Helpers.EnsureBot(&model.Bot{
```

Comment:
Nit: Why not call this `appErr`?
```suggestion
	botUserID, appErr := p.Helpers.EnsureBot(&model.Bot{
```